### PR TITLE
Fredmansky

### PR DESCRIPTION
**Pull request by navro**
F-sektionen **_needs_** this!!!

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -945,6 +952,7 @@
   add_foreign_key "event_users", "events"
   add_foreign_key "event_users", "users"
   add_foreign_key "events", "contacts"
+  add_foreign_key "fredmanskies", "users"
```
[the remark this answers is not in the document]
Comment (navro) [reply db/schema.rb]: Auto generated file, so STFU

File: config/schedule.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+fredmansky_job:
+    cron: "00 15 * 1-5,9-12 FRI"
```
Comment (olofenglund96) [inline config/schedule.yml]: Do we want to be notified exactly at 15.00 or just before? We might want the heads up. Or be able to choose? Might be too advanced though..

Comment (navro) [reply config/schedule.yml]: Good idea! I think we need to have a Fredmansky meeting to discuss this

**Comment by navro** [comment]
> Fredmans always brings out people's best ideas, like this. Love it! I do agree with Olof's comment about the notification time. Also, will there be an option to turn this off for all the boring people?

The boring people will **not** have to find an easter egg to active Fredmansky.

**Comment by FredrikLastow** [comment]
Oh yeah, that's right! What will the easter egg be? 🍻

**Comment by FredrikLastow** [comment]
Nevermind... That's obviously classified

**Comment by navro** [comment]
> Nevermind... That's obviously classified

Yep

**Comment by FredrikLastow** [comment]
> Oh yeah, that's right! What will the easter egg be? 🍻

#OttoTeaser

**Comment by navro** [comment]
I'm going to merge this now, because I want to see if it works tomorrow! We can change the timing in a later commit

**Pull request merged**